corehq/apps/export/models/new.py

```diff
@@ -0,0 +1,191 @@
+from itertools import groupby
+from couchdbkit import SchemaListProperty, SchemaProperty
+
+from corehq.apps.userreports.expressions.getters import NestedDictGetter
+from corehq.apps.app_manager.dbaccessors import get_built_app_ids_for_app_id
+from corehq.apps.app_manager.models import Application
+from dimagi.utils.couch.database import iter_docs
+from dimagi.ext.couchdbkit import (
+    Document,
+    DocumentSchema,
+    ListProperty,
+    StringProperty,
+    IntegerProperty,
+)
+
+from ..const import FORM_TABLE
+
+
+class ExportItem(DocumentSchema):
+    """
+    An item for export.
+    path is a question path like ["my_group", "q1"] or a case property name
+    like ["date_of_birth"].
+    """
+    path = StringProperty()
+    label = StringProperty()
+    last_occurrence = IntegerProperty()
+
+
+class ExportColumn(DocumentSchema):
+    item = SchemaProperty(ExportItem)
+    name = StringProperty()
+    path = StringProperty()
```

Comment:
What's the difference between this `path` and `ExportItem.path`? Same for `label`